examples/error-handling/src/main.rs

```diff
@@ -45,8 +45,9 @@ use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 async fn main() {
     tracing_subscriber::registry()
         .with(
-            tracing_subscriber::EnvFilter::try_from_default_env()
-                .unwrap_or_else(|_| "example_error_handling=debug,tower_http=debug".into()),
+            tracing_subscriber::EnvFilter::try_from_default_env().unwrap_or_else(|_| {
+                concat!(env!("CARGO_CRATE_NAME"), "=debug,tower_http=debug").into()
```

Comment:
I'm afraid that `concat!(env!("CARGO_CRATE_NAME"), "=debug,tower_http=debug")` is much less clear compared to:

`format!("{}=debug,tower_http=debug", env!("CARGO_CRATE_NAME"))`

I do not believe there will be performance issues for our usage scenario.

Reply:
Yeah, it's a bit ugly. Feel free to change it back.

Reply:
Personally, I'd say having
```rust
let crate_name = env!("CARGO_CRATE_NAME");
format!("{crate_name}=debug,tower_http=debug")
```
would be a more readable alternative, but feel free to ignore this, either is fine.